src/zjs_promise.c

```diff
@@ -113,6 +116,7 @@ void zjs_make_promise(jerry_value_t obj, zjs_post_promise_func post,
     new->post = post;
     new->then_set = 0;
     new->catch_set = 0;
+    new->then = jerry_acquire_value(obj);
```

Comment:
It seems like this acquired object will not be released, because "then_set" is checked and this gets overwritten later.